### DISCUSSION TITLE
What’s the best way to keep script up-to-date?

### DISCUSSION THREAD

**Discussion by mikey32230** [KBlixt/subcleaner#4] [Q&A] [answered]
Just curious, what the best way to keep this script up-to-date with any changes you make, as you make them?

Thank you

**Comment by KBlixt** [comment, marked as answer]
you could set up a cron job that pull the latest every day or look into some auto update with git script. don't have to much experience with that though so i don't know. :)

**Reply by mikey32230** [reply]
Sounds good!
Also thanks for making and sharing this script, it’s awesome :)

**Reply by KBlixt** [reply]
thanks :D
let me know of any false positives or any other issues and I'll take a look at it to improve the script for everyone (Y)
